### PR TITLE
CI : add codecov

### PR DESCRIPTION
**Pull request by ad2ien**
- Add codecov to test workflow
- update badges

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -8,6 +8,7 @@ license = "BSD 3-Clause License"
 [tool.poetry.dependencies]
 python = "^3.9"
 pytest = "^7.1.2"
+pytest-cov = "^3.0.0"
```
Comment (SonnevilleDamien) [inline pyproject.toml:11]: I think it's "dev-dependencies", isn't it?

Comment (ad2ien) [reply pyproject.toml]: Right, I moved `pytest` as well.

**Review by ccamel — APPROVED** [review]

**Review by SonnevilleDamien — APPROVED** [review]

**Review by bdeneux — APPROVED** [review]

**Review by amimart — APPROVED** [review]
Nice 👍

**Review by MarcoBousseau — APPROVED** [review]

**Comment by ad2ien** [comment]
@amimart I still don't have the rights to merge, I affect this PR to you

**Pull request merged**